src/components/Articles.jsx

```diff
@@ -6,8 +6,8 @@ import { getArticles, getUsers } from "../../api.get"
 
 export const ArticleList = () => {
```

Comment:
there isnt anything currently visible on the home page until you click on home, you dont see any articles